playground/docker-compose.fork.yml

```diff
@@ -163,5 +167,28 @@ services:
         - ETH_RPC_URL=http://127.0.0.1:8545
     ports:
       - 8001:80
+
+  chain-check:
+    image: curlimages/curl:latest
+    depends_on:
+      - chain
+    command: >
+      sh -c "while true; do
```

Comment:
I had to use `sh -c` explicitly because, for some reason, the container exists right after the startup with the explicitly provided entry point.